Automattic/wp-calypso
# Pricing page rework: Add features list for Security and Complete bundles

#### Proposed Changes

* Add features list for Security and Complete bundles
* Fixes the mobile breakpoint. Slack discussion: p1662027454376939-slack-C03RCHNVC0H
* Fixes some stylelints in related `.scss` files

#### Testing Instructions

1. Do any one of these
    * Click on Jetpack Cloud live link below and wait for the redirect to complete, then goto `/pricing?flags=jetpack/pricing-page-rework-v1`
    * or boot up this PR 
        * Run `git fetch && git checkout add/jp-pricing-rework/featured-product-cards`
        * Run `yarn start-jetpack-cloud`
        * Goto [http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1](http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1)
2. Confirm that featured and simple product cards work fine for both Products and Bundles
3. Confirm that you see the list of features under Security and Complete featured cards
4. Confirm that the data is correct with regards to what is included in which plan
5. Check with smaller screens (<480px)
6. Confirm that you see the foldable "View all features" cards
7. Confirm that the design matches what is defined in Figma

#### Pre-merge Checklist






Completes: 1202796695664022-as-1202796695664129/f


<img width="1248" alt="Screenshot 2022-09-01 at 6 38 30 PM" src="https://user-images.githubusercontent.com/18226415/187921664-daad9879-2801-4629-881b-7f6ab562526a.png">
<img width="527" alt="Screenshot 2022-09-01 at 6 38 46 PM" src="https://user-images.githubusercontent.com/18226415/187921678-d871cfd5-887e-43d7-be3f-c8aa47a93c50.png">
<img width="440" alt="Screenshot 2022-09-01 at 6 39 04 PM" src="https://user-images.githubusercontent.com/18226415/187921683-219e5808-0558-45e7-90d6-4841c7d724c3.png">
<img width="438" alt="Screenshot 2022-09-01 at 6 39 13 PM" src="https://user-images.githubusercontent.com/18226415/187921689-c4c6d9ed-0e50-4877-ba7d-eea19a8d9b0d.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202796695664129
